src/EditorFeatures/Test/ValueTracking/CSharpValueTrackingTests.cs

```diff
@@ -31,16 +32,19 @@ public void SetS(string s)
 }
 ";
             using var workspace = TestWorkspace.CreateCSharp(code);
-            var initialItems = await GetTrackedItemsAsync(workspace);
 
             //
             // property S 
             //  |> S = s [Code.cs:7]
             //  |> public string S { get; set; } [Code.cs:3]
             //
-            Assert.Equal(2, initialItems.Length);
-            ValidateItem(initialItems[0], 7);
-            ValidateItem(initialItems[1], 3);
+            await ValidateItemsAsync(
+                workspace,
+                itemInfo: new[]
+                {
+                    (7, "s"),
+                    (3, "public string S { get; set; } = \"\""),
```

Comment:
honestly, i'd almost prefer your comment form above to this.

Reply:
I didn't want to expend time writing a parser for this at the moment, but I'm not opposed to doing that as a follow up prior to merging to main